dask_geopandas/core.py

```diff
@@ -343,6 +344,48 @@ def hilbert_distance(self, p=15):
 
         return distances
 
+    def morton_distance(self, p=15):
+
+        """
+        Calculate the distance of geometries along the Morton curve
+        
+        The Morton distance can be used to spatially partition Dask-GeoPandas objects,
+        by mapping two-dimensional geometries along the Morton space-filing curve.
+        
+        Each geometry is represented by the midpoint of its bounds and linked to the 
+        Morton curve. The function returns a distance from the beginning 
+        of the curve to the linked point.
+        
+        Morton distance is more performant than ``hilbert_distance`` but can result in 
+        less optimal partitioning.
+
+        The Morton curve is also known as Z-order https://en.wikipedia.org/wiki/Z-order
```

Comment:
```suggestion
        The Morton curve is also known as Z-order https://en.wikipedia.org/wiki/Z-order.

        The Morton distance can be used to spatially partition Dask-GeoPandas objects,
        by mapping two-dimensional geometries along the Morton space-filing curve.
        
        Each geometry is represented by the midpoint of its bounds and linked to the 
        Morton curve. The function returns a distance from the beginning 
        of the curve to the linked point.
        
        Morton distance is more performant than ``hilbert_distance`` but can result in 
        less optimal partitioning.
```

(I think it's good to "drop" the alternative name in the beginning for visibility)